addon/pods/components/frost-list/component.js

```diff
@@ -3,6 +3,25 @@ import Ember from 'ember'
 import _ from 'lodash/lodash'
 import layout from './template'
 
+function findElementsInBetween (array, firstElement, lastElement) {
```

Comment:
I understand the "hit a match, start pushing, hit the second, stop pushing" behavior you're going for here, but I do think this needs a bit of rework in terms of variable names to make the code clearer